samtranslator/model/connector/connector.py

```diff
@@ -208,39 +226,26 @@ def _get_resource_role_name(
     return ref(logical_id)
 
 
-def _get_resource_queue_url(logical_id: str, resource_type: str) -> Optional[Dict[str, Any]]:
-    if resource_type == "AWS::SQS::Queue":
-        return ref(logical_id)
-    return None
+def _get_resource_queue_url(properties: Dict[str, Any]) -> Optional[Any]:
+    return properties.get("Outputs", {}).get("Url")
```

Comment:
Since we're using a ambiguous "properties" anyway, perhaps make it `outputs`, so we don't need to keep repeating the fallback logic?

Reply:
updated.